client/src/controllers/scores/scores_controller.js

```diff
@@ -65,9 +86,26 @@ class ScoresController {
 		})
 	}
 
+	errorScores(scores) {
+		scores = scores || this._scores || []
+		let duplicateErrors = this.duplicateScores(scores)
+
+		let self = this
+
+
+		let otherErrors = scores.filter(score =>
+			typeof score.teamNumber != "number" || score.round == null ||
+			(!this._loading && this._teamNumberList.indexOf(score.teamNumber) === -1)

```

Comment:
You could just use `includes`. It's new to JS and easier to read. As in `this._teamNumberList.includes(score.teamNumber)`